### PR TITLE
Added include and exclude params for coverage reports (issue #6)

### LINKED ISSUE #6: Add support for include_objects and exclude_objects parameters
It could be a comma separated list:
```
-include=obj1,obj2,obj3
-exclude=obj1,obj2,obj3
```
Ex:
`utplsql run ut3/ut3 -exclude=betwnstr,award_bonus`

### PR DESCRIPTION
**Pull request by JustEdro**
Fixes #6

### CODE REVIEW

**Comment by JustEdro** [comment]
Someone pls fix the builds

File: src/main/java/org/utplsql/cli/RunCommand.java [hunk 1]
```diff
@@ -143,15 +177,23 @@ public int run() throws Exception {
...
                         .addReporterList(reporterList)
                         .sourceMappingOptions(sourceMappingOptions[0])
                         .testMappingOptions(testMappingOptions[0])
                         .colorConsole(this.colorConsole)
                         .failOnErrors(true)
-                        .skipCompatibilityCheck(skipCompatibilityCheck)
-                        .run(conn);
+                        .skipCompatibilityCheck(skipCompatibilityCheck);
+
+                for (String includeObject: finalIncludeObjectsList){
```
Comment (pesse) [inline src/main/java/org/utplsql/cli/RunCommand.java]: This should be done in Fluent interface manner to stay within the same pattern. Will need a change in Java API. I will try to do this soon.

Comment (pesse) [reply src/main/java/org/utplsql/cli/RunCommand.java]: Thanks for your efforts, though! Very appreciated!

Comment (JustEdro) [reply src/main/java/org/utplsql/cli/RunCommand.java]: Do you mean create a function like "includeObjects" or something like that? I also can do that, in general, if you are busy now.

Comment (pesse) [reply src/main/java/org/utplsql/cli/RunCommand.java]: Yes, creating a function in the TestRunner class, return the TestRunner instance so we can continue to use fluent interface pattern. The need to create a loop outside just to add objects feels uncomfortable to me :)

**Comment by pesse** [comment]
I recreated the PR here to get around the travis problems: https://github.com/utPLSQL/utPLSQL-cli/pull/58

**Pull request closed without merging**